src/renderers/common/Renderer.js

```diff
@@ -1120,6 +1144,7 @@ class Renderer {
 	dispose() {
 
 		this.info.dispose();
+		this.backend.dispose();
```

Comment:
This throws a runtime error right now.

Reply:
Ah yes I forgot to cleanup my dispose changes. It should be fix with the latest change. I tried using `about:gpucrash` on Chrome on both backends.